server/src/main/java/org/eclipse/openvsx/RegistryAPI.java

```diff
@@ -1716,4 +1716,13 @@ public ResponseEntity<String> getPublicKey(
 
         return ResponseEntity.notFound().build();
     }
-}
\ No newline at end of file
+
+    @GetMapping(path = "/api/version", produces = MediaType.APPLICATION_JSON_VALUE)
+    public ResponseEntity<RegistryVersionJson> getServerVersion() {
+        try {
+            return ResponseEntity.ok(local.getRegistryVersion());
```

Comment:
You can add a `Cache-Control` header here, e.g. `.cacheControl(CacheControl.maxAge(10, TimeUnit.MINUTES).cachePublic())`

Reply:
Done